specs/DefaultDownloadDialog.md

```diff
@@ -0,0 +1,272 @@
+        COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_LEFT;
+    UINT32 defaultPadding = 20;
+    COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_PADDING padding = {
+        defaultPadding, defaultPadding};
+    CHECK_FAILURE(m_webView2_6->put_DefaultDownloadDialogPosition(
+        targetPosition));
+    CHECK_FAILURE(m_webView2_6->put_DefaultDownloadDialogPadding(
+        padding));
+}
+```
+
+```c#
+// Subscribe to the `IsDefaultDownloadDialogOpenChanged` event to make
+// changes in response to the default download dialog opening or
+// closing. For example, if the dialog is anchored to a button in the
+// application, the button can be updated to appear selected when the
+// dialog is opened.
+webView.CoreWebView2.IsDefaultDownloadDialogOpenChanged += delegate (
+    object sender, object e)
+{
+    if (webView.CoreWebView2.IsDefaultDownloadDialogOpen)
+    {
+        // Here the developer can make updates in response to
+        // the default download dialog opening.
+    }
+    else
+    {
+        // Here the developer can make updates in response to the
+        // default download dialog closing.
+    }
+};
+
+void ToggleDownloadDialogCmdExecuted(object target, ExecutedRoutedEventArgs e)
+{
+    if (webView.CoreWebView2.IsDefaultDownloadDialogOpen)
+    {
+        webView.CoreWebView2.CloseDefaultDownloadDialog();
+    }
+    else
+    {
+        webView.CoreWebView2.OpenDefaultDownloadDialog();
+    }
+}
+
+private void SetDefaultDownloadDialogPosition()
+{
+    uint defaultPadding = 20;
+    CoreWebView2DefaultDownloadDialogPosition newPosition =
+        CoreWebView2DefaultDownloadDialogPosition.TopLeft;
+    CoreWebView2DefaultDownloadDialogPadding padding;
+    padding.x = defaultPadding;
+    padding.y = defaultPadding;
+    webView.CoreWebView2.DefaultDownloadDialogPosition = newPosition;
+    webView.CoreWebView2.DefaultDownloadDialogPadding = padding;
+}
+```
+
+# API Details
+
+
+```c#
+[v1_enum]
+/// The default download dialog can be aligned to any of the WebView corners
+/// by setting the `DefaultDownloadDialogPosition` property. The default
+/// position is top-right corner.
+typedef enum COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION {
+
+  /// Top-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_LEFT,
+
+  /// Top-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_TOP_RIGHT,
+
+  /// Bottom-left corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_LEFT,
+
+  /// Bottom-right corner of the WebView.
+  COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION_BOTTOM_RIGHT,
+
+} COREWEBVIEW2_DEFAULT_DOWNLOAD_DIALOG_POSITION;
+
+/// The padding is the distance between the WebView corner specified by the
+/// `DefaultDownloadDialogPosition` and the default download dialog corner
+/// nearest to it. It is measured in raw pixels or logical pixels, depending on
+/// the COREWEBVIEW2_BOUNDS_MODE. Use (0, 0) to align the dialog to the WebView
```

Comment:
Is the bounds mode set on the CoreWebView2Controller? The controller isn't accessible when using WPF/WinForms/WinUI. So the caller won't know the value or know how to change it. Is this an issue?

Reply:
I misunderstood this, changing to say the point should be in the client coordinate space of the WebView